scripts-dev/complement.sh

```diff
@@ -46,4 +46,4 @@ if [[ -n "$1" ]]; then
 fi
 
 # Run the tests!
-COMPLEMENT_BASE_IMAGE=complement-synapse go test -v -tags synapse_blacklist,msc2946,msc3083 -count=1 $EXTRA_COMPLEMENT_ARGS ./tests
+COMPLEMENT_BASE_IMAGE=complement-synapse go test -tags msc2716 -v -count=1 ./tests/main_test.go ./tests/msc2716_test.go
```

Comment:
All Complement tests passing for my own reference, https://github.com/matrix-org/complement/pull/68

```
$ COMPLEMENT_DIR=../complement ./scripts-dev/complement.sh
...
--- PASS: TestBackfillingHistory (11.16s)
    --- PASS: TestBackfillingHistory/parallel (0.00s)
        --- PASS: TestBackfillingHistory/parallel/Unrecognised_prev_event_ID_will_throw_an_error (0.50s)
        --- PASS: TestBackfillingHistory/parallel/Normal_users_aren't_allowed_to_backfill_messages (0.93s)
        --- PASS: TestBackfillingHistory/parallel/Backfilled_historical_events_without_m.historical_come_down_/sync (1.09s)
        --- PASS: TestBackfillingHistory/parallel/Backfilled_historical_events_resolve_with_proper_state_in_correct_order (1.75s)
        --- PASS: TestBackfillingHistory/parallel/Backfilled_historical_events_with_m.historical_do_not_come_down_/sync (1.82s)
        --- PASS: TestBackfillingHistory/parallel/Historical_messages_are_visible_on_federated_server (2.81s)
PASS
```